frontend/src/views/camp/Invitation.vue

```diff
@@ -142,16 +142,14 @@ export default {
 
     if (this.variant === 'default') {
       // Content of api response depends on authenticated user --> reload every time this component is mounted
-      this.invitation?.()
-        .$reload()
-        .then(
-          () => {
-            this.invitationFound = true
-          },
-          () => {
-            this.invitationFound = false
-          }
-        )
+      this.invitation.$reload().then(
```

Comment:
`this.inviation` could be null, no? In the props-list it says it is not required.

Reply:
Fixed in [805bd05](https://github.com/ecamp/ecamp3/pull/5025/commits/805bd05730582f1c9a13ae41bb7e6a91e62f7312)